### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by carlsonsantana** [wapyce/test#2]
The user @admin validate your site template "https://pages-themes.github.io/dinky/" and found these accessibility errors:
* Page https://pages-themes.github.io/dinky/:
  * Issue *a54230f8-6534-4dd8-ad67-8963bf82552a*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.49:1. Recommendation: change background to #f9f9f9.
    * **Selector**: `html > body > div > section > div:nth-child(11) > div > pre > code > span:nth-child(4)`
    * **Context**: `<span class="no">Dependencies</span>`
  * Issue *1a6f6f6c-e349-4d46-85bf-8562db5646d3*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.49:1. Recommendation: change background to #f9f9f9.
    * **Selector**: `html > body > div > section > div:nth-child(11) > div > pre > code > span:nth-child(2)`
    * **Context**: `<span class="no">GitHubPages</span>`
  * Issue *39b41a04-2cfc-40d2-834a-50cf3218d7cd*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.72:1. Recommendation: change text colour to #737362.
    * **Selector**: `html > body > div > section > div:nth-child(11) > div > pre > code > span:nth-child(1)`
    * **Context**: `<span class="c1"># Ruby code with syntax highlig...</span>`
  * Issue *c6dbde6a-100b-41d1-b14b-ad4db88a3437*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.72:1. Recommendation: change text colour to #737362.
    * **Selector**: `html > body > div > section > div:nth-child(10) > div > pre > code > span:nth-child(1)`
    * **Context**: `<span class="c1">// Javascript code with syntax ...</span>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
